contribute/style-guides/documentation-style-guide.md

```diff
@@ -168,3 +168,29 @@ Two words if used as a verb, one word if used as a noun.
 
 * Set up the workspace.
 * Initial setup might take five minutes.
+
+### Inclusive language
```

Comment:
I think this section should be an h2 and above word usage. 

Also, periods should be inside the quotation marks in this section, not outside.